lambda/script.py

```diff
@@ -0,0 +1,141 @@
+import json
+import os
+from typing import Dict, List, Optional
+
+import boto3  # type: ignore
+import googleapiclient.discovery  # type: ignore
+from google.oauth2 import service_account  # type: ignore
+
+client = boto3.client("ssm")
+
+
+def get_scope(scope: Optional[str]) -> List[str]:
+    """
+    Returns the scope from AWS SSM.
+    """
+    return [client.get_parameter(Name=scope, WithDecryption=True)["Parameter"]["Value"]]
```

Comment:
`Option[str]` implies that `scope` can be `None` - is that right?

Reply:
that is right - we're passing in os.environ.get(parameter) so that will return either a string param or None.

Reply:
Cool, and `get_parameter()` doesn't blow up with a `None` for `Name`?

Reply:
So instead of using os.environ.get(param) we've switched to os.environ[param] so that the whole thing blows up if it's not set to avoid this problem